__tests__/app.test.js

```diff
@@ -255,3 +255,32 @@ describe(" get users", () => {
   });
 });
 
+describe("get articles by topic", () => {
+  test("200: should get all articles by topic", () => {
+    return request(app)
+     .get("/api/articles?topic=mitch")
+     .expect(200)
+     .then(({ body }) => {
+        body.allArticles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: "mitch",
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });

```

Comment:
One final really important test that should be added is for a topic that exists but has no associated articles. You will need to have a look at the data to see which topic this is.